src/braket/pennylane_plugin/braket_device.py

```diff
@@ -605,6 +606,61 @@ def execute_and_gradients(self, circuits, **kwargs):
         res = res[0] if len(res) == 1 and active_jac else res
         return res, jacs
 
+    def _is_single_qubit_01_frame(self, f):
+        if self._device.arn == "arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy":
+            return "drive" in f
+        else:
+            raise RuntimeError(
+                f"Single-qubit drive frame for pulse control not defined for "
+                f"device {self._device.arn}"
+            )
+
+    def _is_single_qubit_12_frame(self, f):
+        if self._device.arn == "arn:aws:braket:eu-west-2::device/qpu/oqc/Lucy":
+            return "second_state" in f
+        else:
+            raise RuntimeError(
+                f"Single-qubit drive frame for pulse control not defined for "
+                f"device {self._device.arn}"
+            )
+
+    def _get_frames(self, filter):
+        return {
+            f: info
+            for f, info in self._device.properties.pulse.dict()["frames"].items()
+            if filter(f)
+        }
+
+    @property
+    def settings(self):
+        frames = self._get_frames(filter=self._is_single_qubit_01_frame)
+        frames_12 = self._get_frames(filter=self._is_single_qubit_12_frame)
```

Comment:
Can we give them more specific names? Something like `drive_frames_01` and `drive_frames_12`